packages/itinerary-body/src/util.ts

```diff
@@ -157,18 +157,20 @@ export function getPlaceName(
     companies
   );
   if (
+    // Don't ever show this useless OTP default string
+    place.name === "Default vehicle type" ||
     (place.name.match(/-/g) || []).length > 3 ||
     company?.overridePlaceNames
   ) {
-    if (company && intl) {
+    if (intl) {
       return intl.formatMessage(
         {
           defaultMessage: defaultMessages["otpUi.AccessLegBody.vehicleTitle"],
           description: "Formats rental vehicle company and type",
           id: "otpUi.AccessLegBody.vehicleTitle"
         },
         {
-          company: company.label,
+          company: company?.label,
```

Comment:
What happens if company is undefined? Do we still want to display this label?

Reply:
I think it falls back gracefully